<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Four-Player Chess Analysis Board</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="1" rowspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <widget class="QWidget" name="moveListTab">
       <attribute name="title">
        <string>Move List</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QPushButton" name="nextMoveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="lastMoveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="prevMoveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="firstMoveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QTableWidget" name="moveListTable">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgnTab">
       <attribute name="title">
        <string>PGN4</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <widget class="QPushButton" name="downloadPgnButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(180, 180, 180);</string>
          </property>
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPlainTextEdit" name="pgnField">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="fenGroupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>FEN4</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="1">
        <widget class="QPushButton" name="pasteFenButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(180, 180, 180);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Paste</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="copyFenButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(180, 180, 180);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Copy</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QPlainTextEdit" name="fenField">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="boardGroupBox">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Board</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QPushButton" name="boardResetButton">
         <property name="statusTip">
          <string>Reset board</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(180, 180, 180);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>View:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="playerViewSelect">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yellow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="4">
     <widget class="QWidget" name="view" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>700</width>
        <height>700</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Game"/>
    <addaction name="actionOpen_PGN4"/>
    <addaction name="actionSave_PGN4_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy_FEN4"/>
    <addaction name="actionPaste_FEN4"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionRotate_Board_Left"/>
    <addaction name="actionRotate_Board_Right"/>
    <addaction name="actionFlip_Board"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_PGN4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open PGN4...</string>
   </property>
  </action>
  <action name="actionSave_PGN4_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save PGN4 as...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy_FEN4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy FEN4</string>
   </property>
  </action>
  <action name="actionPaste_FEN4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Paste FEN4</string>
   </property>
  </action>
  <action name="actionRotate_Board_Left">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Board Left</string>
   </property>
  </action>
  <action name="actionRotate_Board_Right">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rotate Board Right</string>
   </property>
  </action>
  <action name="actionFlip_Board">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Flip Board</string>
   </property>
  </action>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="statusTip">
    <string>Start new game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
